code/datums/components/food/decomposition.dm

```diff
@@ -23,12 +23,12 @@
 	/// Used for examining
 	var/examine_type = DECOMP_EXAM_NORMAL
 
-/datum/component/decomposition/Initialize(mapload, decomp_flags = NONE)
+/datum/component/decomposition/Initialize(mapload, semi_preserved_food, decomp_flags = NONE)
```

Comment:
these vars are awful. they dont describe anything. What the hell is a semi_preserved food?

Reply:
It's explained in `code/game/objects/items/food/_food.dm`:
```
///Food that needs to be picked up in order to decompose.
var/semi_preserved_food = FALSE
```

Reply:
my point being that the name should be self-explanatory.

Reply:
I couldn't come up with a better name for the var, what would you recommend for something like that?

Reply:
There, I think that variable better explains it.